packages/jazz-tools/src/coValues/coMap.ts

```diff
@@ -610,6 +610,22 @@ const CoMapProxyHandler: ProxyHandler<CoMap> = {
             }
         }
     },
+    has(target, key) {
+        if (key in target) {
+            return true;
+        }
+
+        if (typeof key === "string") {
```

Comment:
Could you make this more similar to the implementations of the other proxy traps - i.e. handle the case where it's a string key and in the schema and otherwise use `return Reflect.has(...)`?